### PR TITLE
Python: OS X doesn't have librt, so don't link to it.

### PR DESCRIPTION
**Pull request by atombender**
This should the compilation problem mentioned in [issue 1195](https://github.com/grpc/grpc/issues/1195).

### CODE REVIEW

File: src/python/src/setup.py [hunk 1]
```diff
@@ -49,8 +50,9 @@
 _EXTENSION_LIBRARIES = (
     'grpc',
     'gpr',
-    'rt',
 )
+if not "darwin" in sys.platform:
+    _EXTENSION_LIBRARIES += ('rt')
```
Comment (soltanmm) [inline src/python/src/setup.py]: Use `('rt',)` to ensure it's a tuple, not a string expression.

**Comment by atombender** [comment]
@soltanmm: Thanks, fixed.

**Comment by soltanmm** [comment]
Sorry - I forgot to merge this. Could you please merge with the current master and push so that we may merge this back into master?

**Comment by atombender** [comment]
No problem. Done.

**Pull request merged**